lte/gateway/python/magma/mobilityd/tests/uplink_gw_tests.py

```diff
@@ -0,0 +1,43 @@
+"""
+Copyright (c) 2020-present, Facebook, Inc.
+All rights reserved.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree. An additional grant
+of patent rights can be found in the PATENTS file in the same directory.
+"""
+
+
+import logging
+import unittest
+from collections import defaultdict
+import subprocess
+
+from magma.mobilityd.uplink_gw import UplinkGatewayInfo
+
+LOG = logging.getLogger('mobilityd.def_gw.test')
+LOG.isEnabledFor(logging.DEBUG)
+
+
+class DefGwTest(unittest.TestCase):
+    """
+    Validate default router setting.
+    """
+
+    def test_gw_ip_for_DHCP(self):
+        gw_store = defaultdict(str)
+        dhcp_gw_info = UplinkGatewayInfo(gw_store)
```

Comment:
You can pull this repeated setup logic from here into the `setUp` function that can be overridden

Reply:
ok.